hpack/table.py

```diff
@@ -23,6 +23,12 @@ def table_entry_size(name, value):
     return 32 + len(name) + len(value)
 
 
+class _HeaderNameSearchResult(object):
+    def __init__(self):
+        self.min = None
+        self.positions = {}
```

Comment:
Sorry, this was mostly just a question for myself and the other maintainers: it shouldn't affect your pull request. :grin: